cmd/drone-downstream/config.go

```diff
@@ -33,25 +33,6 @@ func settingsFlags(settings *plugin.Settings) []cli.Flag {
 			EnvVars:     []string{"PLUGIN_TOKEN", "DRONE_TOKEN", "DOWNSTREAM_TOKEN"},
 			Destination: &settings.Token,
 		},
-		&cli.BoolFlag{
-			Name:        "wait",
-			Usage:       "Wait for any currently running builds to finish",
-			EnvVars:     []string{"PLUGIN_WAIT"},
-			Destination: &settings.Wait,
-		},
-		&cli.DurationFlag{
-			Name:        "timeout",
-			Value:       time.Duration(60) * time.Second,
-			Usage:       "How long to wait on any currently running builds",
-			EnvVars:     []string{"PLUGIN_WAIT_TIMEOUT"},
-			Destination: &settings.Timeout,
-		},
```

Comment:
Believe @bradrydzewski 's initial ask was that `wait` and `timeout` would apply to the triggered build so should probably modify that accordingly.

Reply:
OK, will try.

Reply:
See @bradrydzewski ''s comments below at https://github.com/drone-plugins/drone-downstream/pull/73#issuecomment-700909665